content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -225,9 +225,8 @@ than one installation for the same tool since different jobs may need different
 versions of the same tool.
 
 The pre-compiled "Default" option calls whatever is already installed on the
-agent and exists in the machine PATH, but this will return a failure if the
-tool was not already installed and its location was not added to the PATH
-system variable.
+agent and exists in the machine PATH, but this returns a failure if the tool is not
+installed and its location was not added to the PATH system variable.
```

Comment:
```suggestion
The pre-compiled "Default" option calls whatever is already installed on the agent and exists in the machine PATH. 
This returns a failure if the tool is not installed and its location was not added to the PATH system variable.
```
rearrange sentences and ensure that each sentence is its own line of code (based on full stop .)